Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino

```diff
@@ -139,16 +141,16 @@ void set_freq(){
   arg = SCmd.next();
   frequency = atof(arg);
   if (arg != NULL){
-    if(frequency > 902 && frequency < 928){
+//    if(frequency > 902 && frequency < 928){
```

Comment:
If It isn't necessary, it could be deleted

Reply:
the change was made